main.tf

```diff
@@ -49,6 +49,7 @@ module "lambda" {
       PERMISSION_POLICY = var.role_permission_policy
       TRUST_POLICY_JSON = var.trust_policy_json
       LOG_LEVEL         = var.log_level
+      LOCALSTACK_HOST   = var.localstack_host
```

Comment:
That's LOCALSTACK_HOSTNAME, not LOCALSTACK_HOST.

Reply:
Oh, right. But that is demonstrating the same thing I'm saying, that we don't need this at all in the terraform module since we don't use `LOCALSTACK_HOST` in the lambda code. We're already using `LOCALSTACK_HOSTNAME`.

Reply:
I'll have to go back and check, but we're setting LOCALSTACK_HOST to "localstack" or "localhost".  I believe LOCALSTACK_HOSTNAME is set to "localstack" or an empty string.

Reply:
If `LOCALSTACK_HOSTNAME` is empty, I'd have to think either the lambda is not running in localstack, or something is off about the docker setup. I was trying to trace that code in the localstack repo, and I _think_ it should end up with the same value as `MAIN_CONTAINER_NAME` or the IP address of the localstack docker container, either of which should work if the docker networking is setup to allow it.

Still, we're not using `LOCALSTACK_HOST` in the lambda code, so there is no need to pass it as an env to the lambda.

Reply:
The variable no longer exists in main.tf.